manifests/service.pp

```diff
@@ -42,20 +42,32 @@
         }
     }
 
+  if $zookeper::config::exhibitor_managed == 'false' {
```

Comment:
This isn't necessary. If exhibitor is managing the zookeeper service then none of the resources in this class should be used. It's probably better to modify the `if` statement in init.pp (https://github.com/opentable/puppet-zookeeper/blob/master/manifests/init.pp#L158) to be:
`if ($manage_service) and (! $exhibitor_managed) {`

Reply:
@travees @alex-roman : are we good to go with this?

Reply:
@MaximeDevalland @travees sure!

Reply:
I have a puppet-modules PR related to this. 
https://github.com/opentable/puppet-modules/pull/14625